specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2257,6 +2257,11 @@
             "modelAsString": true
           },
           "description": "Set the minimum TLS version to be permitted on requests to storage. The default interpretation is TLS 1.0 for this property."
+        },
+        "isNfsV3Enabled": {
```

Comment:
Does this only need be support in create account?
If it will be return when get/list account, should also be added to account properites.
And please update example for it.

Besides that, when will the server be ready for this change?

Reply:
added it to the storage account properties. Thanks!

Reply:
the change is already available in EUAP and few other regions